lib/vfs/src/lib.rs

```diff
@@ -6,6 +6,12 @@ use std::path::{Path, PathBuf};
 use thiserror::Error;
 use tracing::debug;
 
+#[cfg(all(not(feature = "host_fs"), not(feature = "mem_fs")))]
+compile_error!("At least the `host_fs` or the `mem_fs` feature must be enabled. Please, pick one.");
+
+#[cfg(all(feature = "host_fs", feature = "mem_fs"))]
+compile_error!("The `host_fs` and `mem_fs` features are mutually exclusive.");
```

Comment:
I don't think we need this.
Also, people can use both mem_fs and host_fs together in the same implementation (think of mounting two together on a unionfs)... so we need to delete this.

Reply:
Agreed. Resolved!